src/styles/blocks/header/page.scss

```diff
@@ -0,0 +1,22 @@
+html,
+body,
+main,
+head,
+section,
+a,
+div,
+h1,
+h2,
+h3,
+nav,
+ul,
+li,
+p,
+img {
+  margin: 0;
+  padding: 0;
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  letter-spacing: 0;

```

Comment:
You don't need to specify font properties to all these tags, just to html or body.
[STYLES] - Remember to use fallback fonts - alternative font-family in case the main one doesn't work [like this](https://www.w3schools.com/cssref/pr_font_font-family.asp)
![image](https://github.com/mate-academy/layout_catalog/assets/63243136/01c067f9-400a-4a6d-8bad-37c1e14175f4)

Also, it looks like a reset file but if is inside "blocks/header" directory and called as "page.scss}, why?